modules/servicetemplate/src/main/java/org/eclipse/xpanse/modules/servicetemplate/ServiceTemplateManage.java

```diff
@@ -493,6 +494,10 @@ public void reviewServiceTemplateRequest(UUID requestId, ReviewServiceTemplateRe
             throw new ReviewServiceTemplateRequestNotAllowed(
                     "Service template request is not allowed to be reviewed.");
         }
+        if (StringUtils.isEmpty(review.getReviewComment())) {
+            throw new ReviewCommentValueMissingException(
```

Comment:
This is unnecessary. and please set the filed `REVIEW_COMMENT` in `ServiceTemplateRequestHistoryEntity`  to nullable=false

Reply:
removed and updated.